src/fenced/fenced/fence.py

```diff
@@ -68,13 +68,16 @@ def load_disks(self):
             # Only care about da and nvd devices
             if not i.startswith(('da', 'nvd')):
                 continue
+
             # You can pass an "--exclude-disks" argument to fenced
             # to exclude disks from getting SCSI reservations.
             # fenced is called in 12+ with the exclusion flag
             # because the newer generation M-series are going to
             # have NVMe boot drives
-            if i in self._exclude_disks:
-                continue
+            for j in self._exclude_disks:
```

Comment:
I dont understand what is changing here? Seems like its essentially the same thing but less performant?

Reply:
argsparse is passing a string like this `ada0,ada1` on systems with SATA DOMs.

if the disk is `da0` and the DOM is `ada0` the `if i in "ada0,ada1"` was met and is actually excluding `da0`.

Reply:
This doesnt explain this change, it will be exactly the same thing just written in different python.

Reply:
It does though. `self._excluded_disks` was a single line string of `ada0,ada1`.

>>> ed = 'ada0,ada1'
>>> i = 'da0'
>>> i in ed
True

Reply:
You already handled this earlier doing the split.
This change in `for` loop is a no-op.

Reply:
Sorry, I thought you were talking about the `self._exclude_disks` attribute.